contracts/DolaFlashMinter.sol

```diff
@@ -0,0 +1,98 @@
+//SPDX-License-Identifier: None
+
+pragma solidity ^0.8.0;
+
+import "./interfaces/IERC3156FlashBorrower.sol";
+import "./interfaces/IERC3156FlashLender.sol";
+import "./utils/Ownable.sol";
+import "./utils/Address.sol";
+import "./ERC20/IERC20.sol";
+import "./ERC20/SafeERC20.sol";
+
+contract DolaFlashMinter is Ownable, IERC3156FlashLender {
+    using SafeERC20 for IERC20;
+    event FlashLoan(address receiver, address token, uint256 value);
+    event FlashLoanRateUpdated(uint256 oldRate, uint256 newRate);
+    event TreasuryUpdated(address oldTreasury, address newTreasury);
+
+    IERC20 public constant dola =
+        IERC20(0x865377367054516e17014CcdED1e7d814EDC9ce4);
+    address public treasury = 0x926dF14a23BE491164dCF93f4c468A50ef659D5B;
+    uint256 public flashMinted;
+    uint256 public flashLoanRate = 0.0008 ether;
```

Comment:
Is this 0.08%? I think lower would be nice!!!

Reply:
Governance can update the fee rate later on